chromium_src/components/content_settings/core/common/cookie_settings_base.cc

```diff
@@ -119,6 +121,10 @@ bool BraveIsAllowedThirdParty(
       return true;
   }
 
+  if (base::FeatureList::IsEnabled(content_settings::kBraveEphemeralStorage)) {
```

Comment:
This change still makes no sense to me. If you want to allow all 3rd-party cookies, this is not the way to do it. This is a allow list, not a block list and this is going to break things for cookies.

Reply:
This method has nothing to do with DOM storage, BraveContentSettingsManagerDelegate::AllowStorageAccess controls access to dom storage

Reply:
I believe that method calls `ShouldStoreState` which eventually calls `ContentSettingsManagerImpl::AllowStorageAccess` which calls this method. This means that the access list here also applies to DOM Storage.

In any case, my thought here is that the flag can be independent of enabling third-party cookies for now. In this setup testing ephemeral storage would require turning on the ephemeral storage flag and also enabling third-party cookies from settings.

Reply:
This should no longer be a concern because we aren't going to selectively enable storage when the flag is one yet.